packages/opentelemetry-core/src/context/propagation/HttpTraceContext.ts

```diff
@@ -41,19 +47,33 @@ const VERSION = '00';
  *     For more information see {@link https://www.w3.org/TR/trace-context/}
  */
 export function parseTraceParent(traceParent: string): SpanContext | null {
-  const match = traceParent.match(VALID_TRACE_PARENT_REGEX);
+  const trimmed = traceParent.trim();
```

Comment:
Arguable if we need this. Don't believe there's anything in the spec saying we need to support such but also seems nice to have.

That being said, it does add an additional string creation.

Reply:
Isn't it superfluous? I don't think this is being done in other languages. Would be nice to know the reasoning behind this.

Reply:
Basically, there's a chance you could have a trailing whitespace character (OWS) in the traceparent and this removes that, so as not to lump it into the flags validation. It is just a potential nice to have. 

It does look like trailing whitespace values are removed as of Node [10.19.0](https://github.com/nodejs/node/releases/tag/v10.19.0), 12.15.0 and 13.8.0 for security reasons (all released on 02/06/2020), so it will be wasted effort as of those versions. I'm assuming we'll drop Node 8 support at some point to match Node itself, but that will continue to have issues. 

Happy to remove in this PR, or a follow-up, if we don't want that code in there.